integration-tests/pages/homepage/homepagePage.js

```diff
@@ -24,6 +24,10 @@ const homepagePage = () =>
     soc: () => cy.get('[data-test="soc"]'),
     feedbackBanner: () => cy.get('[data-test="feedback-banner"]'),
     sendLegalMail: () => cy.get('[data-test="send-legal-mail"]'),
+    welcomePeopleIntoPrison: () => cy.get('[data-test="welcome-people-into-prison"]'),
+    welcomePeopleIntoPrisonTitleText: () => cy.get('[data-test="welcome-people-into-prison"] h2'),
+    welcomePeopleIntoPrisonLink: () => cy.get('[data-test="welcome-people-into-prison"] a'),
+    welcomePeopleIntoPrisonParagraphText: () => cy.get('[data-test="welcome-people-into-prison"] p'),
```

Comment:
meh.. would model this as a nested group of attributes:

```
welcomePeopleIntoPrison: () => {
    tile: () => cy.get('[data-test="welcome-people-into-prison"]'),
    title: () => cy.get('[data-test="welcome-people-into-prison"] h2'),
    link: () => cy.get('[data-test="welcome-people-into-prison"] a'),
    description: () => cy.get('[data-test="welcome-people-into-prison"] p'),
}
```

Just think how horrible this would be otherwise with `5 x number of tiles` fields here mucking up the place.

Reply:
(both ways are not a very efficient way of retrieving elements but meh.. same style we use elsewhere )

Reply:
changes made